client/src/components/UploadPhoto/UploadPhoto.tsx

```diff
@@ -0,0 +1,50 @@
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+
+import { Box, Button } from '@material-ui/core';
+import { useStyles } from './useStyles';
+
+export default function RecipeReviewCard() {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <Box
+        className={classes.card}
+        display="flex"
+        flexDirection="column"
+        justifyContent="space-around"
+        alignContent="center"
+        alignItems="center"
+      >
+        <Typography className={classes.fontWeight900} variant="h5">
+          Profile Photo
+        </Typography>
+        <CardMedia className={classes.media} image="/1.jpeg" title="Paella dish" />
+        <Typography className={`${classes.opacity} ${classes.width12}`}>
+          Be sure to use a photo that clearly shows your face
+        </Typography>
+        {/* TODO I added 'name=images' att to input which shoud match the name in config Multer package   */}
```

Comment:
Remove this comment here :smiley:

Reply:
I think this is a fine comment to leave in, I guess would make it a `NOTE` instead of a `TODO`. Maybe move it outside of the JSX though unsure if this improves anything 🤔